config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
-  resources :games
+  get '/', to: 'users#index'
+  get '/users/:id', to: 'users#singleuser'
+  get '/users/:id/posts', to: 'users#userposts'
+  get '/users/:id/posts/:id', to: 'users#singlepost'
```

Comment:
- [ ] We don't need all these other routes since we only need to implement only two routes, the `index` and `show` routes for both models. We can do that like so:

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

Reply:
thank you for your constructive feedback